src/server/bitops_family_test.cc

```diff
@@ -22,6 +28,159 @@ using absl::StrCat;
 
 namespace dfly {
 
+class Bytes {
+  using char_t = std::uint8_t;
+  using string_type = std::basic_string<char_t>;
+
+ public:
+  enum State { GOOD, ERROR, NIL };
+
+  Bytes(std::initializer_list<std::uint8_t> bytes) : data_(bytes.size(), 0) {
+    // note - we want this to be like its would be used in redis where most significate bit is to
+    // the "left"
+    std::copy(rbegin(bytes), rend(bytes), data_.begin());
+  }
+
+  explicit Bytes(unsigned long long n) : data_(sizeof(n), 0) {
+    FromNumber(n);
+  }
+
+  static Bytes From(unsigned long long x) {
+    return Bytes(x);
+  }
+
+  explicit Bytes(State state) : state_{state} {
+  }
+
+  Bytes(const char_t* ch, std::size_t len) : data_(ch, len) {
+  }
+
+  Bytes(const char* ch, std::size_t len) : Bytes(reinterpret_cast<const char_t*>(ch), len) {
+  }
+
+  explicit Bytes(std::string_view from) : Bytes(from.data(), from.size()) {
+  }
+
+  static Bytes From(RespExpr&& r) {
+    if (r.type == RespExpr::STRING) {
+      return Bytes(ToSV(r.GetBuf()));
+    } else {
+      if (r.type == RespExpr::NIL || r.type == RespExpr::NIL_ARRAY) {
+        return Bytes{Bytes::NIL};
+      } else {
+        return Bytes(Bytes::ERROR);
+      }
+    }
+  }
+
+  std::size_t size() const {
+    return data_.size();
+  }
+
+  operator std::string_view() const {
+    return std::string_view(reinterpret_cast<const char*>(data_.data()), size());
+  }
+
+  std::ostream& print(std::ostream& os) const {
```

Comment:
`print` -> Print? 
also, please pull the implementation out of the class declaration

Reply:
ok

Reply:
rename the function to the correct code style (starts with capital)